code/controllers/subsystem/garbage.dm

```diff
@@ -1,40 +1,44 @@
 SUBSYSTEM_DEF(garbage)
 	name = "Garbage"
 	priority = 15
-	wait = 20
-	flags = SS_POST_FIRE_TIMING|SS_BACKGROUND|SS_NO_INIT
```

Comment:
PreInit runs on every subsystem, regardless of SS_NO_INIT, you can keep the flag

Reply:
I had at one point created an init proc, but ended up not not needing it. forgot to re-add no-init